src/www/ui/api/documentation/openapi.yaml

```diff
@@ -70,12 +71,22 @@ paths:
         description: Required details for generating new token
         required: true
         content:
-          application/json:
-            schema:
-              $ref: '#/components/schemas/TokenRequest'
```

Comment:
Any reason to remove `application/json` requests?

Reply:
I felt there is no such specific need of following the JSON encoding as all the sent parameters are in the form of string.